_pages/attributes.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: User attributes
 lead: >
-  login.gov user accounts are either proofed (LOA3) or not (LOA1), corresponding to <a href="http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63-2.pdf">NIST 800-63-2</a> levels of assurance (LOA).
+  login.gov user accounts are either proofed (IAL2) or (IAL1), corresponding to <a href="http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63-3.pdf">NIST 800-63-3</a> Identity Assurance Level (IAL).
```

Comment:
please add "self-asserted" in front of "(IAL1)"

Reply:
Great idea! Just updated.